modules/lily-compiler/src/main/java/io/github/tomboyo/lily/compiler/icg/OasSchemaToAst.java

```diff
@@ -74,15 +79,74 @@ public static Fqn fqnForRef(PackageName basePackage, String ref) {
 
   private Pair<Fqn, Stream<Ast>> evaluateSchema(
       PackageName currentPackage, SimpleName name, Schema<?> schema) {
-    // - log that we couldn't generate the thing
-    // - documentation on the stub class
-    // - generate empty stub class so that references to the FQN from other classes still work
+
+    /*
+     * OneOf composed schemas evaluate to a sealed interface. The schema may
+     * be composed of new or existing schemas; in either case, we emit
+     * AddInterface modifiers so that those AST are updated to
+     * implement the new sealed interface before rendering.
+     */
+    if (schema instanceof ComposedSchema c && c.getOneOf() != null) {
```

Comment:
Needs factoring

Reply:
Also we should document that we generate all insline types into the same package because of sealed iface requirements.

Reply:
Oh, this is documented in the test case. I think I'm okay with that, the test cases are the spec more or less.